CHANGELOG

```diff
@@ -1,2 +1,77 @@
-1.1.2
-- Changed to nested module style see: https://github.com/bitcrowd/rubocop-bitcrowd/pull/4
+# Rubocop Bitcrowd Changelog
+
+Presented in reverse chronological order.
+
+## master
+
+https://github.com/bitcrowd/rubocop-bitcrowd/compare/v1.2.1...HEAD
+
+* Put high-level summary here
```

Comment:
I'd suggest to write this in italics, just to make it easier distinguishable from the actual bullet points.

Reply:
Oh. It is meant to be a bullet point :) in the hopes that people continue with bullet points. trying to convince people to be consice and not write essays here

Reply:
Yes, makes absolutely sense. Should have been more on point with my comment 🙈 I meant make it **both** -> a bullet point and somehow highlighted, so that people can see where to put things.

Could e.g. be like this:
```markdown
* Some old point
* *^^^ put high level summary here ^^^*
```